Casks/m/mochi-diffusion.rb

```diff
@@ -9,6 +9,7 @@
 
   auto_updates true
   depends_on macos: ">= :ventura"
```

Comment:
Also looks like this requires macOS 14
```suggestion
  depends_on macos: ">= :sonoma"
```